services/parser-database/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Flask
+firebase-admin
+utils
+https://github.com/timClicks/slate/archive/master.zip
```

Comment:
Does this actually work?

Reply:
apparently it doesn't https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source

Reply:
Yes it works, lol